wasp/src/main/java/com/orhanobut/wasp/NetworkStack.java

```diff
@@ -7,5 +7,5 @@
 
   <T> void invokeRequest(RequestCreator requestCreator, InternalCallback<T> waspCallback);
 
-  <T> T invokeRequest(RequestCreator requestCreator) throws Exception;
+  Object invokeRequest(RequestCreator requestCreator) throws Exception;
```

Comment:
That makes sense to have Object type here, now I'm thinking why we put T in the first place which doesn't make any sense.